MACOSX.md

```diff
@@ -153,7 +153,14 @@ Install Ruby gem dependencies
 Install:
 
 ```
-$ gem install whimsy-asf bundler mail listen
+sudo gem install whimsy-asf mail listen
```

Comment:
I think whimsy-asf should not be installed

Reply:
I removed bundler from that line because it is installed later.
Can you explain why ask-whimsy should be removed.

Reply:
Because the gem is not automatically kept up to date.
But this can be left for a separate update if necessary.

Reply:
See also discussion on private@ about "...old whimsy Gem" for background.  Since whimsy.a.o auto-deploys this whole project, for the common use on the server we're just using the code directly, not the gem.  And the whole 'gem' is really only useful for our server.  Thus, we don't need to install the gem, since it's config is often out of date compared to the source tree.

Reply:
I've removed asf-whimsy gem install from the instructions.